### DISCUSSION TITLE
Does v6 have support for websockets?

### DISCUSSION THREAD

**Discussion by MichaelNZ85** [adonisjs/core#4406] [Help]
I was told that websocket support was coming to v6,  but I can't see anything in the docs. Is it part of AdonisJS core, or a package like Lucid that needs to be added?

**Comment by muco-rolle** [comment]
Hello, @MichaelNZ85, on the [AdonisJS Twitter page](https://twitter.com/adonisframework/status/1755867882527896019) they have shared how you can set it up within an AdonisJS V6 app. However, there's first-party support for [SSE](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events) through the `transmit` package that should cover most of the real-time features. you can read about it [here](https://adonisjs.com/blog/future-plans-for-adonisjs-6)

**Comment by bruno-py** [comment]
This [tutorial](https://medium.com/@armyabakouan/how-to-configure-socket-io-for-adonisjs-v6-4afbcaa1a465) works like a charm
